internal/core/api.rs

```diff
@@ -759,6 +760,81 @@ pub fn quit_event_loop() -> Result<(), EventLoopError> {
         .quit_event_loop()
 }
 
+/// Spawn a Future to run in the Slint event loop
```

Comment:
I think spawn is the consistent name. I would suggest to use the same terminology as in the rust docs though. Afaics the rust docs don't use the term "run" but say what really happens: the future will - in our case - only be polled in the Slint event loop.